docs/manual/apk_extensions.rst

```diff
@@ -0,0 +1,158 @@
+in the Godot documentation.
+
+Permissions
+~~~~~~~~~~~
+
+If you go ahead with APK expansions, make sure your app has (or prompts for) the required permissions.
+Implementing a comprehensive, reliable downloader and expansion handling
+likely requires additional considerations,
+`see the Android documentation <https://developer.android.com/google/play/expansion-files#Permissions>`_ for some pointers.
+For example, you may need to acquire wake locks while downloading, check if the device is connected via Wi-Fi, or
+check the Wi-Fi state before downloading large amounts of data, and finally save the data to external storage, all of which
+may require permissions.
+
+Read the documentation
+~~~~~~~~~~~~~~~~~~~~~~
+
+The `Android documentation on expansion files <https://developer.android.com/google/play/expansion-files#Permissions>`_
+is worth a read not only for OBBs specifically, but also for any implementation in general. Additionally,
+`the Meta Horizon documentation on expansion files <https://developers.meta.com/horizon/documentation/native/ps-assets/>`_,
+and for Required Assets specifically, the
+`Meta blog post announcing the feature <https://developers.meta.com/horizon/blog/introducing-mobile-dlc-support-in-beta/>`_
+provide helpful information.
+
+Setup export presets
+~~~~~~~~~~~~~~~~~~~~
+
+Consider setting up additional export presets preconfigured to export your expansion files, whether proper
+OBB files (exported via the Android exporter, as described below) or PCK files for use as OBB or Required Asset files,
+using the normal Godot `exporting facilities <https://docs.godotengine.org/en/stable/tutorials/export/exporting_pcks.html>`_.
+You can use export filters and inclusion lists in those export presets to make sure only the intended files are exported
+in each extension file, and that those files are excluded from the APK itself.
+When first setting this up, make sure to double check whether the configuration works as intended
+(potentially using tools such as `GodotPCKExplorer <https://github.com/DmitriySalnikov/GodotPCKExplorer>`_).
+
+App implementation
+~~~~~~~~~~~~~~~~~~
+
+After ensuring (and potentially requesting) any needed permissions, the app should check whether the expansion file
+exists at the expected path at launch. Some verification may make sense here, like comparing the file size, hash or signature,
+or trying to decrypt the data if encryption was used. The distribution platform may also offer ways to do signature,
+vendor or licensing checks specific to that distribution platform.
+
+If any required expansion files are missing, the app needs to check for and download them.
+For Google Play, the `Android documentation <https://developer.android.com/google/play/expansion-files#DownloaderService>`_
+provides a starting point of how to implement such a downloader, some of which may be useful in general.
```

Comment:
I think we can remove this section. The Google play instructions for settings up a downloader require use of the Google play licensing service and would not translate to Meta's process.